website/public/js/controllers/tasksCtrl.js

```diff
@@ -221,7 +224,11 @@ habitrpg.controller("TasksCtrl", ['$scope', '$rootScope', '$location', 'User','N
 
     $scope.buy = function(item) {
       User.user.ops.buy({params:{key:item.key}});
-      $rootScope.playSound('Reward');
+      //Plays sound for built in rewards (armor, potions etc)
+      if(item.value <= User.user.stats.gp){
+        $rootScope.playSound('Reward');
+    }
```

Comment:
Oops, you're right!  Clearly I was coding too long yesterday - I'll fix now and recommit/push.